ccd-definition/AuthorisationCaseField/CareSupervision/court-admin.json

```diff
@@ -2085,6 +2085,34 @@
     "UserRole": "caseworker-publiclaw-courtadmin",
     "CRUD": "R"
   },
+  {
```

Comment:
this need to be replicated for the other users. would follow what's been done for the other users (i-e judges...)

Reply:
Ah yes, you're absolutely right. Adding now